pkg/netpol/connlist/connlist.go

```diff
@@ -536,6 +559,10 @@ func (ca *ConnlistAnalyzer) getConnectionsList(pe *eval.PolicyEngine, ia *ingres
 
 	excludeIngressAnalysis := (ia == nil || ia.IsEmpty())
 
+	// validate focus-direction
+	if err := ValidateFocusDirectionValue(ca.focusDirection); err != nil {
```

Comment:
is this the first place where this can be validated?

Reply:
no it is not,
if you run it from command-line, the validate would be done in the `PersistentPreRunE` of list command (see cli/list.go)
but i have added it here too, in case someone is using the API from a "script" for example;
this func `getConnectionsList` is the common for all the API funcs (i.e. `ConnlistFromDirPath`, `ConnlistFromResourceInfos` and `ConnlistFromK8sClusterWithPolicyAPI`)

Reply:
so maybe move it to the beginning of this function?

Reply:
done